packages/expo-updates/utils/src/createManifestForBuildAsync.ts

```diff
@@ -3,13 +3,13 @@ import {
   exportEmbedAssetsAsync,
 } from '@expo/cli/build/src/export/embed/exportEmbedAsync';
 import { drawableFileTypes } from '@expo/cli/build/src/export/metroAssetLocalPath';
+import Server from '@expo/metro/metro/Server';
+import type { BundleOptions } from '@expo/metro/metro/shared/types.flow';
```

Comment:
should we try to resolve `expo -> `@expo/metro`? same as the others like `@expo/metro-config`, not sure whether it didn't break pnpm isolated mode from the past. cc @byCedric

Reply:
yes we can merge this and follow up later if that's a problem. still wonder whether it doesn't break from the past haha

Reply:
I think it’s fine, we’ll keep expo metro versioned by Expo SDK